### DISCUSSION TITLE
Show total number of scores submitted to leaderboard in ranked

### DISCUSSION THREAD

**Discussion by iamateacup** [ppy/osu#31029] [Ideas] [answered]
For example next to the users top play, when hovering over their rank it could show 2004/140050 , or even a percentage so that players can get a general idea of how good the play is, as a rank 300 is very different in a map with 1000 leaderboard submissions vs 10k

**Comment by iamateacup** [comment]
To quickly expand on how this could also be useful, it could also show the number of scores submitted in the local leaderboard aswell, so that you would also be able to see your total number of plays for local songs

**Comment by wwawaawwa** [comment]
Was going to make another thread for this, found this while doing another search before posting

>In stable, a personally submitted score to global beatmap leaderboards will show "\#\<your rank\> of \#\<all beatmap scores\>" under a beatmap's leaderboard. Similar behavior should be implemented in lazer to match stable. I propose that it be added to the hover tooltip that currently shows when the ranking is >=1000, and change the tooltip to show when hovering over *any* card's rank, including the ones in the top 50. This could also be done in the beatmap end screen.
> 
> Rank >=1000
> ![image](https://github.com/user-attachments/assets/6949f739-9908-447b-bf78-8b082abe2ed5)
> 
> Rank <1000
> ![image](https://github.com/user-attachments/assets/c02bb57d-2df1-42e7-9a09-76f07898adca)
> 
> Stable
> ![image](https://github.com/user-attachments/assets/fc6162f6-7eda-4822-86cf-30454268e143)
> 
> Quick Mockup
> ![image](https://github.com/user-attachments/assets/6b9f8e5a-6424-4eae-92dc-b945222c63ff)

**Reply by iamateacup** [reply]
To expand on this, i looked at the osu api to see if this data was publicly avaliable, but I wasn't able to find the total number of ranked scores avaliable on the api. There was the total playcount avaliable but this resulted in duplicate scores, and i tried to just call every leaderboard score and count the number of returns, which was successful for maps with under 50 plays, but i was unable to ever see the top 50 scores. I don't have much coding knowledge, so would happily like to see if i missed something though.

**Reply by peppy** [reply]
The data for this would need to be returned as part of the personal best query. It is provided by the component which does rank index lookups.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/31145
